Tests/Pages/ResidentPageInElegance.cs

```diff
@@ -0,0 +1,33 @@
+﻿
+using AlineRevenueRMS_QA.Pages;
+using OpenQA.Selenium;
+
+namespace Core.pages
+{
+    public class ResidentPageInElegance : BasePage
+    {
+        private By TaskListInResidentActivity = By.XPath("//div[contains(text(), 'Resident Activity')]/../..//a[@class='dropdown-toggle']");
+        private By ResidentLedgerAdmin = By.LinkText("• Resident Ledger Admin");
```

Comment:
why do you use LinkText?

Reply:
Because nothing another can`t find, I can show in app